packages/v4/patternfly-docs/content/design-guidelines/styles/colors/colors.md

```diff
@@ -87,72 +87,69 @@ import './colors.css';
   </GridItem>
   <GridItem xl={6} lg={12} rowSpan={6}>
     <img width="443px" src="./patternfly-status.png" alt="PatternFly status colors" />
   </GridItem>
-  <GridItem xl={6} lg={12}>
-    <h3>Danger</h3>
-    <ColorSwatch color="--pf-global--danger-color--300" caption="alert icon">
-      This color is used as the icon color for the danger alert component.
-    </ColorSwatch>
-    <ColorSwatch color="--pf-global--danger-color--200" caption="alert title">
-      This color is used as the title color for the danger alert as well as the background color for the danger button.
-    </ColorSwatch>
-    <ColorSwatch color="--pf-global--danger-color--100" caption="alert icon background">
-      This color is most commonly used as an indication of danger or error for components, such as alerts and form inputs.
-    </ColorSwatch>
-  </GridItem>
   <GridItem xl={6} lg={12}>
     <h3>Default</h3>
-    <ColorSwatch color="--pf-global--default-color--300" caption="alert title and icon">
-      This color is used as the icon and title color for the default alert component.
+    <ColorSwatch color="--pf-global--default-color--300" caption="alert title">
+      This color is used as the title color for the default alert component.
     </ColorSwatch>
-    <ColorSwatch color="--pf-global--default-color--200">
-      This color is used as the background and icon color for the default inline alert component.
+    <ColorSwatch color="--pf-global--default-color--200" caption="alert icon">
+      This color is used as the icon color for the default alert component.
     </ColorSwatch>
-    <ColorSwatch color="--pf-global--default-color--100" caption="alert icon background">
-      This color is used as the icon background color for the default alert component.
+    <ColorSwatch color="--pf-global--palette--cyan-50" caption="alert background">
+      This color is used as the background color for the default inline alert component.
     </ColorSwatch>
   </GridItem>
   <GridItem xl={6} lg={12}>
-    <h3>Info</h3>
-    <ColorSwatch color="--pf-global--info-color--200" caption="alert title and icon">
-      This color is used as the icon and title color for the info alert component.
+  <h3>Danger</h3>
+    <ColorSwatch color="--pf-global--danger-color--200" caption="alert title">
+      This color is used as the title color for the danger alert component.
     </ColorSwatch>
-    <ColorSwatch color="--pf-global--info-color--100" caption="alert icon background">
-      This color is used as the icon background color for the info alert component as well as the background and icon color for the info inline alert component.
+    <ColorSwatch color="--pf-global--danger-color--100" caption="alert icon">
+      This color is used as the icon color for the danger alert component.
+    </ColorSwatch> 
+    <ColorSwatch color="--pf-global--palette--red-50" caption="inline alert background">
```

Comment:
This one says "inline alert background" but the "-50" colors for the other statuses do not - should they?